Packs/QRadar/Playbooks/playbook-QRadar_Indicator_Hunting_V2.yml

```diff
@@ -1773,30 +1971,33 @@ tasks:
     note: false
     timertriggers: []
     ignoreworker: false
+    continueonerrortype: ""
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
   '151':
     id: '151'
-    taskid: d2db9183-bbb1-4911-870f-0db9cdd672d8
+    taskid: 19ea0add-8395-4d36-8d32-0d2ae1a371b6
     type: playbook
     task:
-      id: d2db9183-bbb1-4911-870f-0db9cdd672d8
+      id: 19ea0add-8395-4d36-8d32-0d2ae1a371b6
       version: -1
       name: QRadarFullSearch
```

Comment:
Can we already change the name of the task?

Reply:
NP - on it :)